contracts/proxies/SafeProxy.sol

```diff
@@ -31,21 +31,25 @@ contract SafeProxy {
 
     /// @dev Fallback function forwards all transactions and returns all received return data.
     fallback() external payable {
-        // solhint-disable-next-line no-inline-assembly
+        /* solhint-disable no-inline-assembly */
+        /// @solidity memory-safe-assembly
```

Comment:
Given https://github.com/safe-global/safe-contracts/pull/711/files#r1444473657

Should we also revert this change? This is not part of the inheritance graph for `Safe` contract, so it should be fine to have this not be memory safe right?

cc @mmv08

Reply:
Just to be clear, reverting the changes made using `ptr` only or both `ptr` and the `memory-safe-assembly` tag as well?

Reply:
If you revert ptr but keep the memory-safe-assembly tag, then it's going to be memory-unsafe code

Reply:
Yes, done 👍🏾